net/mosdns/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+name                mosdns
```

Comment:
`name` is implicitly set by `go.setup`.  You could remove this line, and move the `go.setup` group of stanzas up here.